toolchains/solidity/core/crates/linter-lib/Cargo.toml

```diff
@@ -2,7 +2,7 @@
 name = "solidhunter-lib"
 description = "Solidhunter/Osmium solidity linter library"
 repository = "https://github.com/astrodevs-labs/osmium"
-version = "0.0.2"
+version = "0.0.3"
```

Comment:
Should be incremented as minor since it is in pre release and this is a breaking change